### PR TITLE
Update README with dockerized development environment

### PR DESCRIPTION
**Pull request by ftarulla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,60 +6,75 @@
...
 
-5. Setup elasticsearch index template: `bundle exec rake elasticsearch:setup`
+5. Setup elasticsearch (we need the environment running):
+	```
+	# environment up
+	$ docker-compose up
 
-6. Run tests: `bundle exec rake` (this will run `rspec` and `cucumber`)
+	# setup elasticsearch index template
+	$ docker-compose run --rm web rake elasticsearch:setup
```
Comment (matiasgarciaisaia) [inline README.md:36]: I'd run a single `docker-compose run --rm web rake db:setup db:test:prepare elasticsearch:setup` as all we care about is having the environment ready to work on it.

File: README.md [hunk 2]
```diff
@@ -6,60 +6,75 @@
...
+	```
 
-7. Start development server: `bundle exec rails s`
+### Additionally setup
 
-Additionally:
-
-8. Import manifests: `bundle exec rake manifests:load`
+1. Import manifests: `bundle exec rake manifests:load`
```
Comment (matiasgarciaisaia) [inline README.md:41]: Are these mandatory? Or are they samples to have the database pre-populated with something? If that's the case, let's state that (it's optional, you may want to run it if you need XXXXX).

File: README.md [hunk 3]
```diff
@@ -6,60 +6,75 @@
...
-		* We support elasticsearch versions < 2.x
-		* Install it in mac with: `brew install elasticsearch17`
+2. Install Docker.
 
-3. Setup development database: `bundle exec rake db:setup`
+3. Install `gems`:
+	```
+	$ docker-compose build
+	$ docker-compose run --rm web gem install bundler -v `<2`
+	$ docker-compose run --rm web bundle install
```
Comment (matiasgarciaisaia) [inline README.md:21]: Do we really need to `gem install bundler`?

File: README.md [hunk 4]
```diff
@@ -108,3 +123,29 @@ Now, whenever a new csv file enters the sshd inbox, it will be imported into the
 ### Sync File Watcher - Client Side
 
 In the client side, you will need to run another filewatcher: [cdx-sync-client](https://github.com/instedd/cdx-sync-client). It is a Windos App. Install it using its NSI installer, restart your computer, and fill the form that will prompt after first restart.  You will be required to provide an activation token - you can generate it form the device manager in the CDP app.
+
+### Old setup (previous to Docker)
```
Comment (matiasgarciaisaia) [inline README.md:127]:
```suggestion
### Legacy setup (local installation, without Docker)
```

**Review by matiasgarciaisaia — COMMENTED** [review]
👍

File: README.md (hunk 2 above)
Comment (ftarulla) [reply README.md:41]: As far as I can see those steps seems to be necessary for working with devices.
What do you think about:
### Additional setup for working with devices

File: README.md (hunk 1 above)
Comment (ftarulla) [reply README.md:36]: Fixed it. It works great!

File: README.md (hunk 3 above)
Comment (ftarulla) [reply README.md:21]: yep, you're right! We don't need it!

**Pull request merged**